riva/client/nmt.py

```diff
@@ -25,6 +37,99 @@ def __init__(self, auth: Auth) -> None:
         self.auth = auth
         self.stub = riva_nmt_srv.RivaTranslationStub(self.auth.channel)
 
+    def streaming_s2s_response_generator(
```

Comment:
Is this called somewhere?

Reply:
I mean how are these utility functions getting tested?

Reply:
called from the notebook https://github.com/nvidia-riva/tutorials/pull/145/files

Reply:
ok.